apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/(analysis)/summary/components/MultipleChoiceSummary.tsx

```
@@ -84,90 +89,94 @@ export const MultipleChoiceSummary = ({
+                onClick={() =>
+                  setFilter(
+                    questionSummary.question.id,
+                    questionSummary.question.headline,
+                    questionSummary.question.type,
+                    questionSummary.type === "multipleChoiceSingle" || otherValue === result.value
+                      ? t("environments.surveys.summary.includes_either")
+                      : t("environments.surveys.summary.includes_all"),
+                    [result.value]
+                  )
+                }>
+                <div className="text flex flex-col justify-between px-2 pb-2 sm:flex-row">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)